ci-requirements.txt

```diff
@@ -4,6 +4,7 @@ isort==4.3.4
 pylint==1.9.3
 autopep8==1.4.3
 httpretty==0.9.6
+pytest==4.0.2
```

Comment:
Fixed pytest version because from 4.1.0 release this issue happens and test fails
https://github.com/pytest-dev/pytest-cov/issues/252
https://github.com/pytest-dev/pytest/pull/4564

Reply:
I see. thanks